guide/inflation.rst

```diff
@@ -151,72 +262,142 @@ In the namelist each entry has two values. The first is for Prior inflation and
-it to be the input for the next assimilation cycle. That is the time to make a copy 
+it to be the input for the next assimilation cycle. That is the time to make a copy
 that has a unique name - usually with some sort of date or timestamp. This also makes
-it possible to restart an experiment.
-
-The suggested procedure for testing inflation options is to start without any (both ``inf_flavor`` values set to 0 and
-``inf_damping`` > 0.). Then enable Prior state space, spatially-varying inflation, with no Posterior inflation (set
-``inf_flavor`` to [2, 0]). Then try damped inflation (set ``inf_damping`` to 0.9 and set ``inf_sd_initial`` and
-``inf_sd_lower_bound`` to 0.6). The inflation values and standard deviation are written out to files with
-``_{prior,post}inf_{mean,sd}{_d##}.nc``, in their names
-(d\#\# is the domain number, if needed). These NetCDF files can be viewed with common tools (we often use
-`ncview <http://meteora.ucsd.edu/~pierce/ncview_home_page.html>`__ ). Expected inflation values are generally in the 1
-to 30 range; if values grow much larger than this it usually indicates a problem with the assimilation.
-
-:doc:`../assimilation_code/programs/fill_inflation_restart/fill_inflation_restart` 
-may be used to create netCDF files with initial values such that the 
-input.nml settings for reading from file vs. reading from namelist can stay constant 
-throughout the entire experiment.
-
-It is possible to set inflation values in an existing netCDF file by using one of the standard NCO utilities like
-"``ncap2``" on a copy of a restart file. Inflation mean and sd values look exactly like restart values, arranged by
-variable type like T, U, V, etc.
-
-Here's an example of using ncap2 to set the T,U and V inf values:
-
-.. container:: unix
-
-   ::
-
-        ncap2 -s 'T=1.0;U=1.0;V=1.0' wrfinput_d01 input_priorinf_mean_d01.nc
-        ncap2 -s 'T=0.6;U=0.6;V=0.6' wrfinput_d01 input_priorinf_sd_d01.nc
-        -or-
-        ncap2 -s 'T(:,:,:)=1.0;U(:,:,:)=1.0;V(:,:,:)=1.0' wrfinput_d01 input_priorinf_mean_d01.nc
-        ncap2 -s 'T(:,:,:)=0.6;U(:,:,:)=0.6;V(:,:,:)=0.6' wrfinput_d01 input_priorinf_sd_d01.nc
-
-Some versions of the NCO utilities change the full 3D arrays into a single scalar. If that's your result (check your
-output with ``ncdump -h``) use the alternate syntax or a more recent version of the NCO tools.
+it possible to restart an experiment. For many of the supported models, the scripting that
+does the renaming and copying of the inflation files may already be provided.
+
+.. Important::
+		Inflation is only applied to the variables that are **updated** by DART
+		by specifying ``UPDATE`` within the ``&model_nml`` section of ``input.nml``.
+		Alternatively, if the variables are not updated (``NO_COPY_BACK``) then inflation
+		will not impact them. ``NO_COPY_BACK`` variables are often used to compute the forward
+		operators, however, they don't take part in the update and as such
+		they will not be inflated.
```

Comment:
@mgharamti @braczka 
A question on this 'important' section. 

NO_COPY_BACK does not prevent the variables being inflated, a user has to set the  initial inflation file to have inflation values of 1.0 to have have no inflation #276
Do you have code elsewhere that queries no_copy_back before applying inflation?

Reply:
Helen, I thought this feature has already been implemented after the science discussions we had. Perhaps, this is something I'll work on in a separate PR (in the near future). We will need to query the variables before applying inflation.
 
For the sake of this documentation PR, I'll edit the text for the `NO_COPY_BACK` section and push.